packages/protovalidate/src/error.ts

```diff
@@ -36,7 +36,7 @@ import { FieldDescriptorProto_Type } from "@bufbuild/protobuf/wkt";
  */
 export class CompilationError extends Error {
   override name = "CompilationError";
-  constructor(message: string, options?: ErrorOptions) {
+  constructor(message: string, options?: { cause?: unknown }) {
```

Comment:
`ErrorOptions` is a ES2022 feature. We provide a cause for informational purposes, but we don't strictly require the runtime to support it. 

We don't want users targeting ES2020 to run into compiler errors (through lib checks), so I'm replacing `ErrorOptions` with a literal type here.